### ISSUE TITLE
latest release 1.19.1 not available for google-beta provider

### ISSUE THREAD

**Issue by gburiola** [terraform-providers/terraform-provider-google#2274]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Terraform Version

Latest release 1.19.1 was only built for the main `google` provider but not for `google-beta`.

https://releases.hashicorp.com/terraform-provider-google/
https://releases.hashicorp.com/terraform-provider-google-beta/

Is that intentional ?

**Comment by wmuizelaar** [comment]
The beta-provider is in the same binary, isn't it?

**Comment by gburiola** [comment]
@wmuizelaar no, it's a separate binary. When I run `terraform init` it tries to download the `google-beta` provider from https://releases.hashicorp.com/terraform-provider-google-beta/ but that address only contains version 1.19.0 and not 1.19.1

**Comment by paddycarver** [comment]
Hi @gburiola! Sorry for the late reply (HashiConf!) and confusion around this. We're still working on our automation for google-beta, and deciding whether we want non-beta resources in the google-beta provider or not. We took a look at the bug fixes involved in 1.19.1, and believed them to not impact users who were using google-beta for just beta resources, so we didn't issue a release.

In general, we are treating the google and google-beta providers as separate providers. Which means they may not have the same version at all times. This is to preserve our ability to issue patch releases when only one of the providers has a problem, or even to only issue major releases if only one of the providers requires it.

We're still evaluating these choices and understanding their impact on our users, so if you have feedback, we'd love to hear it.

In the meantime, can you tell us more about why you expected a google-beta 1.19.1, and what a 1.19.1 release of the google-beta provider would help you to achieve?

**Comment by gburiola** [comment]
Hey @paddycarver , thanks for the update.

I was just trying to keep the version of the providers in sync, so I thought whenever a new version of `google` was release, the would also be a version of `google-beta`.

There was nothing in particular that I was looking in the beta version.

My personal opinion is that to avoid confusion, we should always release both providers at the same time, even if only one of them is changing.

**Comment by paddycarver** [comment]
Hey @gburiola, thanks for the feedback! It's definitely something we'll keep in mind as we continue to discuss and refine our approach to this. Our goal is to make things as straightforward and intuitive for users, but also to not annoy users with multiple versions that are identical, prompting an upgrade for no real reason. This is a bit of new ground for us and for Terraform as a whole, so we're proceeding carefully and with a lot of thought and discussion.

**Comment by paddycarver** [comment]
(I'd look into resolving this immediately, but we feel 2.0.0 is coming soon, and both beta and GA providers will be in sync at that point, so I'm not convinced it's worth the considerable effort required to make it happen. If it is important to someone, please shout, and we can reevaluate.)
